crates/red_knot_python_semantic/resources/mdtest/typed_dict.md

```diff
@@ -0,0 +1,21 @@
+# `TypedDict`
+
+We do not support `TypedDict`s yet. This test mainly exists to make sure that we do not emit any
+errors for the definition of a `TypedDict`.
+
+```py
+from typing_extensions import TypedDict
+
+class Person(TypedDict):
+    name: str
+    age: int | None
+
```

Comment:
I'll add `TODO` for `NotRequired` elements (and `total=False` for the functional syntax)